.ci/pipelines/build.Jenkinsfile

```diff
@@ -41,6 +41,9 @@ pipeline {
                     }
                 }
                 stage('Upload YAML manifest to S3') {
+                    when {
+                        buildingTag()
+                    }
                     steps {
                         script {
                             env.VERSION = readFromEnvFile("VERSION")
```

Comment:
Should `$VERSION`  not point to `$CURRENT_RELEASE + 1` as soon as we branch off for the release and avoid the problem you described?

Reply:
Ideally, yes. However it's a manual process that's not clearly documented so we have not been very good at doing that promptly. 
Even if the `VERSION` file is correct, the generated manifest doesn't point to the correct container image so uploading that to S3 currently doesn't achieve much. I was in the process of creating an issue about publishing the nightly build so I will add this problem to the list of things to fix in that issue.